src/event_handler.py

```diff
@@ -0,0 +1,44 @@
+import boto3
+import json
+import logging
+import os
+
+from base64 import b64decode
+from urllib.request import Request, urlopen
+from urllib.error import URLError, HTTPError
+
+
+# The base-64 encoded, encrypted key (CiphertextBlob) stored in the kmsEncryptedHookUrl environment variable
+ENCRYPTED_HOOK_URL = os.environ['kmsEncryptedHookUrl']
+SLACK_CHANNEL = os.environ['slackChannel']
+
+HOOK_URL = "https://" + boto3.client('kms').decrypt(CiphertextBlob=b64decode(ENCRYPTED_HOOK_URL))['Plaintext'].decode('utf-8')
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+
+def notify_slack(event, context):
+    logger.info(f"Event: {str(event)}")
+    message = json.loads(event['Records'][0]['Sns']['Message'])
+    logger.info(f"Message: {str(message)}")
+
+    alarm_name = message['AlarmName']
+    #old_state = message['OldStateValue']
```

Comment:
😕

Reply:
Weeeeeeeell, you know, just in case . . . :P